integreat_compass/cms/constants/offer_group_types.py

```diff
@@ -0,0 +1,11 @@
+"""
+This module contains all string representations of all group types in which offers
+can be offered, used by :class:`~integreat_compass.cms.models.offers.offer.Offer`
+"""
+from django.utils.translation import gettext_lazy as _
+
+PRIVATE = 0
+GROUP = 1
+BOTH = 2
```

Comment:
```suggestion
PRIVATE = "PRIVATE"
GROUP = "GROUP"
BOTH = "BOTH"
```
Since the underlying model fields are `CharField`s, you have to use strings as choices for them. You could also convert them to integer fields to save a few bytes of db storage, but the advantage of having readable string constants is that it's easier to debug and you don't have to remember the meaning of the numbers when inspecting form values etc...